single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -870,9 +870,10 @@ private void drawFeederInfo(String prefixId, FeederNode feederNode, List<Point>
 
         String svgId = escapeId(feederNode.getId()) + "_" + feederInfo.getComponentType();
         g.setAttribute("id", svgId);
+        String measurementType = feederInfo.getComponentType();
```

Comment:
why "measurement"? The value in feederInfo can be calculated, estimated, and the info can also be anything: a note, an icon. So in fact calling it something else than "componentType" would be misleading.